api/src/certification/session-management/domain/read-models/InvigilatorSession.js

```diff
@@ -0,0 +1,16 @@
+class InvigilatorSession {
+  constructor({ finalizedAt, invigilatorPassword }) {
+    this.invigilatorPassword = invigilatorPassword;
+    this.finalizedAt = finalizedAt;
```

Comment:
Plutot que un getter pourquoi ne pas evaluer directement les falsy et s'eviter un getter ?

```suggestion
    this.isAccessible = !finalizedAt;
```

Reply:
Pourquoi pas, on peut exposer les attributs interne d'un objet après tout.